src/scss/_vertical-slide.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes slideIn {
+  from {
+    opacity: 0;
+    transform: translateY(-10%);
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.sliderParentRow {
```

Comment:
All CSS class names need to be update to use lower case and dashes. No camel case, please.

Reply:
Css was modified.